### PR TITLE
alert dialog implemented

### PR DESCRIPTION
**Pull request by Joe102192**

### CODE REVIEW

**Review by Dalton098 — APPROVED** [review]
qwow awesome sauce thank you joe

File: frontend/Weather101/src/app/app.component.scss [hunk 1]
```diff
@@ -72,7 +72,7 @@
 }
 
 .right-selection-item {
-    justify-content: right;
+    justify-content: flex-end;
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/app.component.scss:75]: the only flex is perhaps the big, qu

File: frontend/Weather101/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -21,7 +22,8 @@ export default function weatherServiceSpy() {
       "latLonHourlyWeatherForcast",
       "getStoredLocation",
       "fetchLocation",
-      "setOfficeLocation"
+      "setOfficeLocation",
+      "activeAlerts"
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/app.component.spec.ts:26]: noetactive enough

File: frontend/Weather101/src/app/app.component.ts [hunk 3]
```diff
@@ -19,18 +20,28 @@ export class AppComponent {
...
-  constructor(private weatherService: WeatherService) {
+  constructor(private weatherService: WeatherService,
+              private dialog: MatDialog) {
     this.searchText = this.weatherService.getStoredLocation().zip;
   }
 
   activeAlerts() {
-    this.weatherService.activeAlerts(this.weatherService.getStoredLocation()).subscribe(data => console.log(data));
+    this.weatherService.activeAlerts(this.weatherService.getStoredLocation()).subscribe(data => {
+      console.log(data);
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/app.component.ts:30]: consider ofquantization?

**Review by JoeBartelmo — APPROVED** [review]
Zbig

**Pull request merged**